x-pack/legacy/plugins/uptime/server/rest_api/overview_filters/get_overview_filters.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { UMServerLibs } from '../../lib/lib';
+import { UMRestApiRouteFactory } from '../types';
+import { objectValuesToArrays } from '../../lib/helper';
+
+const arrayOrStringType = schema.maybe(
+  schema.oneOf([schema.string(), schema.arrayOf(schema.string())])
+);
+
+export const createGetOverviewFilters: UMRestApiRouteFactory = (libs: UMServerLibs) => ({
+  method: 'GET',
+  path: '/api/uptime/filters',
+  validate: {
+    query: schema.object({
+      dateRangeStart: schema.string(),
+      dateRangeEnd: schema.string(),
+      search: schema.maybe(schema.string()),
+      locations: arrayOrStringType,
+      schemes: arrayOrStringType,
+      ports: arrayOrStringType,
+      tags: arrayOrStringType,
+    }),
+  },
+
+  options: {
+    tags: ['access:uptime'],
+  },
+  handler: async ({ callES }, _context, request, response) => {
+    const {
+      dateRangeStart,
+      dateRangeEnd,
+      locations,
+      schemes,
+      search: searchString,
```

Comment:
Feels weird to alias here, but no big deal. Maybe we could call `search` below `parsedSearch` or similar.

Reply:
Modified in https://github.com/elastic/kibana/pull/50837/commits/17bd0a72546d1c30ae0cb684d6a2e80c4b3670d0.